tests/unit/test_client.py

```diff
@@ -80,6 +80,7 @@ def test_request_headers(mock_get_and_post):
             client_info_header: client_info_value,
         },
         redirect_handler=None,
+        role="hive=ALL",
```

Comment:
Please add a test case with multiple roles.

Reply:
It would be nice to add an example in README how to set role.

Reply:
it still misses test for multiple roles

Reply:
2 more added